src/utils.ts

```diff
@@ -104,3 +106,18 @@ export const doSearch = (query: string, registryName: string) =>
 export const getUniqueItemsInArray = (array: any) => {
 	return array.filter((item: any, index: number, self: any) => self.indexOf(item) === index);
 };
+
+export const getNorwegianLabelFirst = (listOfLabels: MultipleLabelType[]): string => {
+	let label: string = EMPTY;
+	if (listOfLabels.length > 1) {
+		listOfLabels.forEach((element: MultipleLabelType) => {
+			if (element.language === 'nb') {
+				label = element.value;
+				return;
+			}
+		});
+	} else {

```

Comment:
Wondering if nb _or_ nn _or_ en should be the first choice and then anything else (we may get, e.g. Finnish in here at some point).